src/components/hub/ChatForm.js

```diff
@@ -23,17 +23,26 @@ import { useSelector, useDispatch } from "react-redux";
 
 import "@styles/hub/ChatForm.css";
 
+import ChatFormContent from "@components/hub/chatForm/ChatFormContent.js";
+import ChatFormHeader from "@components/hub/chatForm/ChatFormHeader.js";
+import ChatFormInputs from "@components/hub/chatForm/ChatFormInputs.js";
+import LockScreen from "@components/hub/LockScreen";
+
 export default function ChatForm() {
   const dispatch = useDispatch();
   const location = useLocation();
 
-  const isUserLogin = useSelector(getUserIsLoggedIn);
-  const isTabInFocus = useSelector(getIsTabInFocus);
-
+  const currentUserId = useSelector(selectCurrentUserId);
   const conversations = useSelector(selectConversationsEntities);
   const selectedConversation = useSelector(getConverastionById);
   const selectedCID = selectedConversation?._id;
 
+  const isUserLogin = useSelector(getUserIsLoggedIn);
+  const isTabInFocus = useSelector(getIsTabInFocus);
+  const [isSuccesESession, setIsSuccesESession] = useState(
```

Comment:
I assume we want to store here a flag which will indicate if it's possible to have e2ee chat in e2ee conv or not

Simply storing `selectedConversation?.is_encrypted` as a default value will confuse cause it can be `true` but in reality it can be some other reason the e2ee will not be possible in this particular case.

let's do it this way 

```
const [successfulEncryptedSession, setSuccessfulEncryptedSession] = useState(null)
```